src/main/java/com/sktTask/ProductService/Sender.java

```diff
@@ -0,0 +1,25 @@
+package com.sktTask.ProductService;
+
+import com.sktTask.ProductService.models.Product;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+@Service
+public class Sender {
+    private static final Logger log= LoggerFactory.getLogger(Sender.class);
+    private final RabbitTemplate rabbitTemplate;
+
+
+    public Sender(RabbitTemplate rabbitTemplate) {
+        this.rabbitTemplate = rabbitTemplate;
+    }
+    @Scheduled(fixedDelay = 3000L)
```

Comment:
This is the sender, so this should be in the `web-app`, should be done when you start structuring your project.